### PR TITLE
Product upload

### PR DESCRIPTION
**Pull request by minju25kim**
- user can upload picture
- user can submit contents of product
- user can be redirected to new product page

### CODE REVIEW

File: components/Products/ProductUpload.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    const cloudinary = await fetch(API_ENDPOINT, options).then((response) => {
+      console.log("picture is uploaded to cloudinary");
+      return response.json();
+    });
+
+    // for image preview
+    setImageSrc(cloudinary.secure_url);
+
+    // insert products input to supabase
+    const { data, error } = await supabase.from("products").insert([
```
Comment (alternadiva) [inline components/Products/ProductUpload.js:44]: Amazing work on the file upload and data insert ✨

File: components/Products/ProductUpload.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+    // upload picture to cloudinary
+    const API_ENDPOINT =
+      "https://api.cloudinary.com/v1_1/streetcred/image/upload";
+    const options = {
+      method: "POST",
+      body: formData,
+    };
+    const cloudinary = await fetch(API_ENDPOINT, options).then((response) => {
+      console.log("picture is uploaded to cloudinary");
```
Comment (sonianb) [inline components/Products/ProductUpload.js]: we might want to remove the `console.log` ?

Comment (minju25kim) [reply components/Products/ProductUpload.js]: it's done

File: components/Products/ProductUpload.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            ref={category}
+            // onChange={(event) => setCategory(event.target.value)}
+          />
+          <label htmlFor="lend">Lend</label>
+
+          <input
+            type="radio"
+            id="pass-on"
+            name="category"
+            value="pass on"
```
Comment (sonianb) [inline components/Products/ProductUpload.js:93]: I think you're missing a dash here?

**Comment by sonianb** [comment]
[Select and display items from the database](https://trello.com/c/p81auZV4/35-select-and-display-items-from-the-database)

**Comment by sonianb** [comment]
[Product page should display the item's info](https://trello.com/c/ldBqC2r5/44-product-page-should-display-the-items-info)

**Pull request merged**